client/human/HumanClientFSM.cpp

```diff
@@ -826,7 +866,7 @@ boost::statechart::result WaitingForGameStart::react(const GameStart& msg) {
 
     Client().GetClientUI().GetPlayerListWnd()->Refresh();
 
-    return transit<PlayingTurn>();
+    context<HumanClientFSM>().process_event(DoneLoading());
```

Comment:
why does this function generate the `DoneLoading` event and not the caller? It and `ProcessData` and `ProcessUI` don't seem to be used elsewhere.